### DISCUSSION TITLE
Add Debug in browser with Fetch?

### DISCUSSION THREAD

**Discussion by ezequidias** [qruto/laravel-wave-client#14] [Ideas]
I'm following the new changes and it's getting amazing!
would it be a good idea to put debug in the eventsource with fetch?

```
window.Echo = new Echo({ broadcaster: WaveConnector, debug: true });

```

in fetcheventsource:
```
fetchEventSource('/api/sse', {
    onmessage(msg) {
        if (options.debug) {
            console.debug('[EventSource]', msg);
        }
```

[fetch event source - issues 3](https://github.com/Azure/fetch-event-source/issues/3)

**Comment by slavarazum** [comment]
Thanks! It was a little bit tricky to migrate all thing especially test cases :)

Yep, we are totally need a sweet debug ability. Still preparing release...

**Reply by ezequidias** [reply]
Thanks!!

**Comment by slavarazum** [comment]
@ezequidias Just published fresh release.

[0.7.0](https://github.com/qruto/laravel-wave/releases/tag/0.7.0)

Cooked some sweet debugging with table mode. Hope it will be useful.

![wave-debug](https://github.com/qruto/laravel-wave-client/assets/5820718/fa9ba521-db41-432c-9d30-46aa8fdebaf0)

Thank you for participating 🤝 Let me know if you will have any issues.

**Reply by ezequidias** [reply]
Laravel wave is getting really awesome! I'm loving all of this! Congratulations @slavarazum
